src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -538,4 +573,46 @@ private Optional<BibEntryTableViewModel> findEntry(BibEntry entry) {
     public void setCitationMergeMode(boolean citationMerge) {
         this.citationMergeMode = citationMerge;
     }
+
+    private void updatePlaceholder(VBox placeholderBox) {
+       if (database.getDatabase().getEntries().isEmpty()) {
+           this.setPlaceholder(placeholderBox);
+       } else {
+           this.setPlaceholder(null);
+       }
+    }
+
+    private BibEntry addExampleEntry() {
+        BibEntry exampleEntry = new BibEntry(StandardEntryType.Article);
+        exampleEntry.withField(StandardField.AUTHOR, "Oliver Kopp and Carl Christian Snethlage and Christoph Schwentker");
+        exampleEntry.withField(StandardField.TITLE, "JabRef: BibTeX-based literature management software");
+        exampleEntry.withField(StandardField.JOURNAL, "TUGboat");
+        exampleEntry.withField(StandardField.VOLUME, "44");
+        exampleEntry.withField(StandardField.NUMBER, "3");
+        exampleEntry.withField(StandardField.PAGES, "441--447");
+        exampleEntry.withField(StandardField.DOI, "10.47397/tb/44-3/tb138kopp-jabref");
+        exampleEntry.withField(StandardField.ISSN, "0896-3207");
+        exampleEntry.withField(StandardField.ISSUE, "138");
+        exampleEntry.withField(StandardField.YEAR, "2023");
```

Comment:
Chain these statemetns
```suggestion
BibEntry exampleEntry = new BibEntry(StandardEntryType.Article)
    .withField(StandardField.AUTHOR, "Oliver Kopp and Carl Christian Snethlage and Christoph Schwentker")
    .withField(StandardField.TITLE, "JabRef: BibTeX-based literature management software")
    .withField(StandardField.JOURNAL, "TUGboat")
    .withField(StandardField.VOLUME, "44")
    .withField(StandardField.NUMBER, "3")
    .withField(StandardField.PAGES, "441--447")
    .withField(StandardField.DOI, "10.47397/tb/44-3/tb138kopp-jabref")
    .withField(StandardField.ISSN, "0896-3207")
    .withField(StandardField.ISSUE, "138")
    .withField(StandardField.YEAR, "2023");
```